apps/cnquery/cmd/scan.go

```diff
@@ -97,6 +97,13 @@ You can also explicitly request the scan of an image or a container registry:
 			"container-image": {
 				Short: "Scan a container image.",
 			},
+			"container-tar": {
+				Short: "Scan a container image from a tar file.",
+				Long: `Scan OCI container image. This supports more parameters for different registries:
```

Comment:
Just wondering about the relevance of the second sentence for tars?  It's talking about registries so maybe it isn't

Reply:
Great catch @imilchev I updated the text.